src/screens/LocationPage/LocationPage.tsx

```diff
@@ -39,33 +33,18 @@ function LocationPage({ region }: LocationPageProps) {
         pageTitle={getPageTitle(region)}
         pageDescription={getPageDescription(region)}
       />
-      <NavBar renderSecondaryElement={() => <DonateButtonHeart />} />
+      <NavLocationPage
+        renderSecondaryElement={() => <DonateButtonHeart />}
+        renderSearch={renderNavBarSearch}
+        menuOpen={menuOpen}
+        setMenuOpen={setMenuOpen}
+        hasScrolled={hasScrolled}
+      />
```

Comment:
Since `NavLocationPage` will always render the same search bar, we could remove the renderNavBarSearch prop and just include `NavBarSearch` in the implementation of `NavLocationPage`?